index.html

```diff
@@ -4,11 +4,195 @@
+            Hackathons which help you put your coding skills to work. Solve
+            interesting business problems and real-world challenges. Register
+            now.
+          </p>
+          <h2 class="hero-text-time">2022.12.01(SUN) ~ 16(FRI)</h2>
+          <h3 class="hero-text-place">@ Exhibition gathering, Cairo, Egypt</h3>
+        </div>
+      </section>
+      <section id="program">
+        <div class="program-title">
+          <h2>You will learn</h2>
+          <div class="underline"></div>
+        </div>
+        <div class="program-cards">
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_01.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_02.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_03.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_04.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_05.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+        </div>
+        <a class="program-call-to-action" href="#">
+          Join Hackathon Global Summit 2022
+        </a>
+      </section>
+      <section id="speakers">
+        <div class="program-title">
+          <h2>Speakers</h2>
+          <div class="underline"></div>
+        </div>
+        <div class="chevron">
+          <div class="speakers-wrapper"></div>
+        </div>
+        <button class="mobile-element show-more-speakers">
+          MORE<span
+            ><img
+              class="chevron-arrow"
+              src="./assets/icons/down-arrow.png"
+              alt="down-arrow"
+          /></span>
+        </button>
```

Comment:
[OPTIONAL] In conformity to standard best practices, the more button should change to less when all the speakers are displayed. 

![Screenshot from 2022-10-19 16-18-48](https://user-images.githubusercontent.com/65233860/196750602-eb51807d-db7e-4795-a77e-3faff0d6eeab.png)